task7/script.js

```diff
@@ -0,0 +1,57 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク"
+}, {
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+}];
+
+const getListItem = () => {
+  return new Promise((resolve) => {
+    resolve(listItem);
+    insertLoading();
+  });
+};
+
+// ローディング画像を表示
+const insertLoading = () => {
+  const loading = document.getElementById("loading");
+  const img = document.createElement("img");
+  img.src = "loading-circle.gif";
+  img.id = "loadingImg";
+  loading.appendChild(img);
+}
+
+getListItem().then((value) => {
+  setTimeout(() => {
+    // ローディング画像を非表示
+    const img = document.getElementById("loadingImg");
+    img.src = "";
```

Comment:
Thank you for the review.
I confirmed on the development tool that the node remains and also understand that unnecessary nodes need to be deleted.

I didn't know the remove method, so I learned it.
commit: 3767ac0

Reply:
```
  // ローディング画像を非表示
  const img = document.getElementById("loadingImg");
  img.remove();
```
IMO
I felt it would be better to make a function here.
Just as there is a function to add loading, I think the process of removing loading can be made into a function.